### ISSUE TITLE
pep8 renamed to pycodestyle

### ISSUE THREAD

**Issue by bernarduh** [msherry/flycheck-pycheckers#66]
During a fresh install of Ubuntu, I noticed that the `pep8` command is now distributed as `pycodestyle`. It would be nice to support this in the `bin/pycheckers.py` scripts.

> This package used to be called pep8 but was renamed to pycodestyle to reduce confusion. Further discussion can be found [in the issue where Guido requested this change](https://github.com/PyCQA/pycodestyle/issues/466), or in the lightning talk at PyCon 2016 by @IanLee1521: [slides](https://speakerdeck.com/ianlee1521/pep8-vs-pep-8) [video](https://youtu.be/PulzIT8KYLk?t=36m).

Quote from https://pypi.org/project/pycodestyle/

I can make the change if you want but what would be the best approach:

- Consider it a new linter class based on pep8.
- Adapt pep8 class to detect which binary is installed in the system.

**Comment by msherry** [comment]
Thanks for submitting the issue! I would prefer that users who are using old tools/versions for whatever reason (their employer hasn't yet completed migrating to py3 yet, for example) still be able to use this tool, so as long as this doesn't break existing compatibility, I would be ok with either of those approaches. Thank you for the offer!

**Comment by bernarduh** [comment]
Yes, I agree. Can I push a branch and open a PR with a suggestion?

**Comment by msherry** [comment]
By all means :)
